src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -1090,10 +1108,42 @@ - (void)_reportAttributes:(NSArray<NSDictionary<NSString *, id> *> *)attributes
     }
 }
 
+// When we receive an attribute report, check if there are any changes in parts list, server list, device type list, accepted commands list,
+//  attribute list, cluster revision or feature map attributes of the descriptor cluster. If yes, make a note that the device configuration changed.
```

Comment:
Again, the "of the descriptor cluster" does not sound right here.

Reply:
fixed.